pkg/cluster/root_app.go

```diff
@@ -60,8 +63,15 @@ func ConstructRootApp(
 		app.ObjectMeta.Labels["arlon-cluster"] = clusterName
 		// assume CAPI for now
 		apiProvider = "capi"
+		if withCAS {
+			err := enableClusterAutoscaler(apiProvider, &helmParams, &ignoreDiffs)
```

Comment:
Instead of passing helmParams and ignoreDiffs by pointer, it'd be more elegant for the function to return new lists:

`helmParams, ignoreDiffs, err := enableClusterAutoscaler(apiProvider, helmParams, ignoreDiffs)`

Inside the function, the new lists are constructed by appending to the input lists. Less copying around this way.